cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -210,14 +217,20 @@ get_docker_default_runtime(){
 }
 
 get_docker_version(){
-	sudo docker version | awk '/Engine/{getline; print $2 }'
+	if [[ "$ID" =~ ^opensuse.*$ ]]; then
+		sudo docker version --format '{{.Client.Version}}' | cut -d '.' -f1-2
+	else
+		sudo docker version | awk '/Engine/{getline; print $2 }'
+	fi
 }
 
 get_docker_package_name(){
 	if [ "$ID" == "ubuntu" ] || [ "$ID" == "debian" ]; then
 		dpkg --get-selections | awk '/docker/ {print $1}'
-	elif [ "$ID" == "fedora" ] || [ "$ID" == "centos" ] || [[ "$ID" =~ ^opensuse.*$ ]] || [ "$ID" == "sles" ]; then
+	elif [ "$ID" == "fedora" ] || [ "$ID" == "centos" ] || [ "$ID" == "sles" ]; then
 		rpm -qa | grep docker | grep -v selinux
+	elif [[ "$ID" =~ ^opensuse.*$ ]]; then
+		rpm -qa | grep docker-$(get_docker_version) | grep -v selinux
```

Comment:
I see that docker has other dependencies that are not removed when uninstalling it, but we should also uninstall them as they may conflict with a newer/older version of docker, e.g.: 
```
fuentess@test-opensuse:~> sudo zypper install 'docker<18.09'
Loading repository data...
Reading installed packages...
Resolving package dependencies...

Problem: docker-18.06.1_ce-lp150.5.6.1.x86_64 requires docker-libnetwork-git = 3ac297bc7fd0afec9051bbb47024c9bc1d75bf5b, but this requirement cannot be provided
  not installable providers: docker-libnetwork-0.7.0.1+gitr2664_3ac297bc7fd0-lp150.3.3.1.x86_64[openSUSE-Leap-15.0-Update]
 Solution 1: Following actions will be done:
  downgrade of docker-libnetwork-0.7.0.1+gitr2711_2cfbf9b1f981-lp150.3.10.1.x86_64 to docker-libnetwork-0.7.0.1+gitr2664_3ac297bc7fd0-lp150.3.3.1.x86_64
  downgrade of containerd-1.2.2-lp150.4.10.1.x86_64 to containerd-1.1.2-lp150.4.6.1.x86_64
  downgrade of docker-runc-1.0.0rc6+gitr3748_96ec2177ae84-lp150.5.14.1.x86_64 to docker-runc-1.0.0rc5+gitr3562_69663f0bd4b6-lp150.5.7.1.x86_64
 Solution 2: do not install docker-18.06.1_ce-lp150.5.6.1.x86_64
 Solution 3: break docker-18.06.1_ce-lp150.5.6.1.x86_64 by ignoring some of its dependencies

```